### PR TITLE
Add webfont for icons

### PR DESCRIPTION
**Pull request by rurseekatze**
Add custom webfont generated with http://fontello.com/. It will be used for icons on a future version of the website. The included `font/config.json` can be used to re-generate the custom font. Some CSS files mentioned in `font/README.txt` were removed because they are not necessary here.

### CODE REVIEW

**Comment by Nakaner** [comment]
If the code is needed for the new version of the web frontend, why don't you create a separate branch for the new frontend and file the pull request *against* that branch?

File: font/LICENSE.txt [hunk 1]
```diff
@@ -0,0 +1,12 @@
+Font license info
+
+
+## Font Awesome
+
+   Copyright (C) 2016 by Dave Gandy
+
+   Author:    Dave Gandy
+   License:   SIL ()
```
Comment (Nakaner) [inline font/LICENSE.txt]: Please add either the link to the license document or the text of the license itself.

**Comment by rurseekatze** [comment]
> If the code is needed for the new version of the web frontend, why don't you create a separate branch for the new frontend and file the pull request against that branch?

I do not understand the problem. I could have created one big pull request for the website redesign. But I wanted to split up the work, so I decided to make a separate pull request for the font. Because the website depends on the font, it does make sense to add the font to the repository before I will create pull requests for the website redesign.

File: font/LICENSE.txt (hunk 1 above)
Comment (rurseekatze) [reply font/LICENSE.txt]: OK, I added a link to the license.

Fontello says on https://github.com/fontello/fontello/wiki/What-about-license%3F:

> If you place content of fontello's archive somewhere with your source code, that's enough. See, example for fontello itself.

**Comment by DerDakon** [comment]
I think this boils down to: this is not needed as it is not used. You could just leave this open until you have the first usage of it ready, this should not cause any collisions so merging it later will likely not cause any problems.

**Pull request closed without merging**